src/pretix/base/settings.py

```diff
@@ -37,6 +38,20 @@ def country_choice_kwargs():
     }
 
 
+class LazyI18nStringList(UserList):
+    def __init__(self, init_list=None):
+        super().__init__()
+        if init_list is not None:
+            self.data = [v if isinstance(v, LazyI18nString) else LazyI18nString(v) for v in init_list]
```

Comment:
I guess this should technically be

```suggestion
            self.data = [LazyI18nString(copy.deepcopy(v.data)) if isinstance(v, LazyI18nString) else LazyI18nString(v) for v in init_list]
```

Or the [initializer of LazyI18nString](https://github.com/raphaelm/django-i18nfield/blob/109de9f1d3c8621dc09e9d5a2095d652f97ba1bd/i18nfield/strings.py#L14) is updated to work with `v` being `dicts`, `str` or another `LazyI18nString` (so it copies the data over). Then this could just be:

```suggestion
            self.data = [LazyI18nString(v) for v in init_list]
```

Reply:
I think since this is mostly a wrapper for serialization/unserialization, we should be fine without the copy for now. Or did you see it cause any problems?

Reply:
We are fine for now. ^^
I guess not copying elements, just the initial list is actually how `list` works, so that's fine.